AutomatedTesting/Gem/PythonTests/Atom/atom_utils/material_editor_utils.py

```diff
@@ -211,7 +211,7 @@ def timed_out(self):
         return time.time() > self.die_after
 
 
-screenshotsFolder = os.path.join(azlmbr.paths.devroot, "AtomTest", "Cache" "pc", "Screenshots")
+screenshotsFolder = os.path.join(azlmbr.paths.projectproductassets, "Screenshots")
```

Comment:
`projectproductassets` is quite verbose, would `projectproducts` (assuming it's generally understood that "products" are assets in the cache) work as well? Or `productassets` or even `products` since products are always created in a project's cache?

Reply:
`products` sounds good.